packages/components/order-tracking/src/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import component from './order-tracking.component';
+
+import './index.scss';
+
+const moduleName = 'orderTracking';
+const componentName = 'orderTrackingComponent';
+
+angular
+  .module(moduleName, [])
```

Comment:
You need at least `oui`, `@ovh-ux/ng-translate-async-loader` and `angular-translate`